app/models/passenger.rb

```diff
@@ -9,6 +9,10 @@ class Passenger < ApplicationRecord
   scope :active, -> { where(active: true) }
   scope :inactive, -> { where(active: false) }
 
+  # TODO: Make configurable by user
+  MOBILITY_DEVICES = ['Boot', 'Crutches', 'Cane', 'Walker',
```

Comment:
This will be getting sorted in a different branch - just going to leave it as a model constant for this branch.